src/sass/themes/beagle.scss

```diff
@@ -112,3 +112,9 @@ $medium-editor-placeholder-color: #f8f5f3;
     text-decoration: none;
   }
 }
+
+.medium-editor-toolbar-actions {
+  li, button {
+    border-radius: 50px;
```

Comment:
You shouldn't use the hardcoded value `50px` but `$medium-editor-border-radius` instead